cooperative-delivery-wg/charter.md

```diff
@@ -0,0 +1,121 @@
+# Cooperative Delivery Working Group Charter
+
+## Interested Parties
```

Comment:
Maybe move this section to the end? Otherwise it is too long to put at the beginning...

Reply:
Done